en/frames_multicopter/dji_f450_cuav_5plus.md

```diff
@@ -0,0 +1,212 @@
+# DJI F450 + CUAV V5+ Build
+
+This topic provides full instructions for building the kit and configuring PX4 using *QGroundControl*.
+
+Key information
+
+- **Frame:** DJI F450
+- **Flight controller:** [CUAV V5+](../flight_controller/cuav_v5_plus.md)
+- **Assembly time (approx.):** 90 minutes (45 minutes for frame, 45 minutes autopilot installation/configuration)
+
+![Finished setup 1](../../assets/airframes/multicopter/dji_f450_cuav_5plus/f450_cuav5_plus_complete.jpg)
+
+## Bill of materials
+
+The components needed for this build are:
+* Power module (Holybro)
+* Fully assembled Power Management Board with ESCs
+* Motors - DR2205 KV2300
+* 5” Plastic Props
+* Carbon fiber 250 airframe with hardware
+* Foxer camera 
```

Comment:
@tony3dr
- Does this build really have a camera and Vtx as shown here? If so, how did they get purchased/where are they from?
- Anything else not really in this build?